src/main/java/com/car/app/model/dto/CarDto.java

```diff
@@ -0,0 +1,80 @@
+package com.car.app.model.dto;
+
+import java.util.Date;
+
+import com.car.app.model.Car;
+
+public class CarDto {
+
+	private String id;
+	
+	private Date createdDate;
+	
+	private Date updatedDate;
+
+	private String brand;
+	
+    private Date registration;
+	
+	private String country;
+
+	public String getId() {
+		return id;
+	}
+
+	public void setId(String id) {
+		this.id = id;
+	}
+
+	public Date getCreatedDate() {
+		return createdDate;
+	}
+
+	public void setCreatedDate(Date createdDate) {
+		this.createdDate = createdDate;
+	}
+
+	public Date getUpdatedDate() {
+		return updatedDate;
+	}
+
+	public void setUpdatedDate(Date updatedDate) {
+		this.updatedDate = updatedDate;
+	}
+
+	public String getBrand() {
+		return brand;
+	}
+
+	public void setBrand(String brand) {
+		this.brand = brand;
+	}
+
+	public Date getRegistration() {
+		return registration;
+	}
+
+	public void setRegistration(Date registration) {
+		this.registration = registration;
+	}
+
+	public String getCountry() {
+		return country;
+	}
+
+	public void setCountry(String country) {
+		this.country = country;
+	}
+	
+	
+	static public CarDto convertCarToDto(Car car) {
```

Comment:
Creating a method in a class to perform the mapping is one approach, to a simple class like this is possible to do it. When we have complex classes with complex structures this approach could not be the best solution.
In that cases there are solutions like:
https://mapstruct.org/
http://modelmapper.org/